Tranga/MangaConnectors/ChromiumDownloadClient.cs

```diff
@@ -14,38 +14,10 @@ internal class ChromiumDownloadClient : DownloadClient
     
     private async Task<IBrowser> DownloadBrowser()
     {
-        BrowserFetcher browserFetcher = new BrowserFetcher();
-        foreach(string rev in browserFetcher.LocalRevisions().Where(rev => rev != ChromiumVersion))
-            browserFetcher.Remove(rev);
-        if (!browserFetcher.LocalRevisions().Contains(ChromiumVersion))
-        {
-            Log("Downloading headless browser");
-            DateTime last = DateTime.Now.Subtract(TimeSpan.FromSeconds(5));
-            browserFetcher.DownloadProgressChanged += (_, args) =>
-            {
-                double currentBytes = Convert.ToDouble(args.BytesReceived) / Convert.ToDouble(args.TotalBytesToReceive);
-                if (args.TotalBytesToReceive == args.BytesReceived)
-                    Log("Browser downloaded.");
-                else if (DateTime.Now > last.AddSeconds(1))
-                {
-                    Log($"Browser download progress: {currentBytes:P2}");
-                    last = DateTime.Now;
-                }
-
-            };
-            if (!browserFetcher.CanDownloadAsync(ChromiumVersion).Result)
-            {
-                Log($"Can't download browser version {ChromiumVersion}");
-                throw new Exception();
-            }
-            await browserFetcher.DownloadAsync(ChromiumVersion);
-        }
-        
```

Comment:
chrome is a part of base image now

Reply:
Could you rename method to "StartBrowser" for readabilitys sake?

Reply:
renamed